packages/solidity-contracts/test/messagesIncomingV3.test.ts

```diff
@@ -595,610 +633,709 @@ describe('FuelMessagePortalV3 - Incoming messages', () => {
     });
   });
 
-  describe('Behaves like V2 - Accounting', () => {
```

Comment:
There is one last issue here, I think you have accidentally introduced some `describe` blocks like `Behaves like V2 - Accounting` inside the `describe` block of `V3 Proxy Initialization`, and it is applying indentation to tests that should remain otherwise unchanged.